src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,68 @@
+package duke.task;
+
+/**
+ * Encapsulates a task object.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String taskType;
+
+    /**
+     * Constructor for this task.
+     *
+     * @param description Details of the task
```

Comment:
Remember to add punctuations after header comment statements too! (Like full-stops here)

Reply:
Right I will add them!